### ISSUE TITLE
Core Lightning is super old

### ISSUE THREAD

**Issue by jaonoctus** [getumbrel/umbrel-apps#330]
## Description

CLN version is outdated. I cannot even decode or pay signet invoices from LND because of it.

![image](https://user-images.githubusercontent.com/7918067/206636797-c55179bc-a496-40a3-bbbc-ca9442dfadfa.png)

## Possible fix

Just ask [lncm ](https://github.com/lncm) folks to build a new image. The current version is 22.11

I built one my self and it's working on a clean installation, but might not work for those that already have a old version running.

```diff
// docker-compose.yml
- image: lncm/clightning:v0.11.0@sha256:75e0ce04d644f34b07bc8a3b92e58b3db4e3c06bdc0e0cecd1669bc3b2d53421
+ image: jaonoctus/cln:v22.11@sha256:f466829730cd46a0d87f0ac367cbb60643d03c669bd5a78f2d1ed178295b57e8
```

**Comment by jaonoctus** [comment]
- Related to lncm/docker-clightning#6

**Comment by nevets963** [comment]
Closing this issue as this PR (https://github.com/getumbrel/umbrel-apps/pull/355) was merged in to update Core Lightning to v22.11.1
